src/python/pants/base/exception_sink.py

```diff
@@ -533,4 +533,4 @@ def _handle_signal_gracefully(cls, signum, signame, traceback_lines):
 # to explicitly request it. The exception log will have any stacktraces regardless so this should
 # not hamper debugging.
 ExceptionSink.reset_should_print_backtrace_to_terminal(
-  should_print_backtrace=os.environ.get('PANTS_PRINT_EXCEPTION_STACKTRACE') == 'True')
+  should_print_backtrace=os.environ.get('PANTS_PRINT_EXCEPTION_STACKTRACE', 'True') == 'True')
```

Comment:
Do you know why this doesn't use `global_options.py`?

https://github.com/pantsbuild/pants/blob/b4e316f4badc2703cea566267886d8b094d0f569/src/python/pants/option/global_options.py#L283-L285

Reply:
Because it runs at import time! This is overwritten immediately after we parse bootstrap options: https://github.com/pantsbuild/pants/blob/b4e316f4badc2703cea566267886d8b094d0f569/src/python/pants/bin/pants_runner.py#L58

Beyond an environment variable, I wasn't sure how to allow the user to affect how errors are displayed at import time. There may be a more reasonable interface I'm not seeing.